src/comparison-operators.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import "fmt"
+
+// Comparison-operators program demonstrates different
+// comparison operators in Go, like:
+// == (equal to)
+// != (not equal to)
+//  < (less than)
+//  > (greater than)
+// <= (less than or equal to)
+// >= (greater than or equal to)
+func main() {
+	x := 3
```

Comment:
Thanks for the useful contribution towards comparison operators.
 It would  add more value to the example if you include comparison of other data types like String, Array, Struct, Boolean  etc.

Reply:
> Thanks for the useful contribution towards comparison operators.
> It would add more value to the example if you include comparison of other data types like String, Array, Struct, Boolean etc.

@aprabhat I am closing the PR for now. let's create separate issue for **suggested changes**.